data/tmpl/utility/content5.html

```diff
@@ -49,7 +49,11 @@
           </li>
         </ul>
       </footer>
+      <div class="prize hidden">
+        <a class="prize">Claim your prize!</a>
```

Comment:
This should be a `button` element, as it does something and not send you somewhere.